### DISCUSSION TITLE
Does this support IE 11?

### DISCUSSION THREAD

**Discussion by bugzpodder** [radix-ui/primitives#1559] [Help] [answered]
Since this only provides unstyled components I would assume there would be no issues with IE 11 if you have the right polyfills?

**Comment by jjenzz** [comment, marked as answer]
Radix doesn't aim for IE11 support so it would be up to you to dig into the implementation details and supply polyfills for things like `ResizeObserver`. However, that seems pretty risky as Radix implementation details could change without warning... they could add a new non-IE11 feature to any component later without you realising.

**Reply by benoitgrelard** [reply]
Jenna is spot on here, I'll just add our `browserslist` support here for more clarity:

https://github.com/radix-ui/primitives/blob/8f75e5be679b9c269638bddf532a166d80edecb4/babel.config.js#L30
